pkg/ddl/schema.go

```diff
@@ -218,6 +219,23 @@ func onDropSchema(jobCtx *jobContext, job *model.Job) (ver int64, _ error) {
 			break
 		}
 
+		// Split tables into multiple jobs to avoid too big transaction.
```

Comment:
Although you split to multiple `asyncNotifyEvent`, DDL still uses the same transaction to commit. I think you means "...avoid too big records"

Reply:
Upatded.